smarts/core/waypoints.py

```diff
@@ -112,7 +112,9 @@ def __eq__(self, other):
         "wp",  # Waypoint: current waypoint
         "nexts",  # list of LinkedWaypoint: list of next immediate waypoints
         # it's a list of waypoints because a path may branch at junctions
+        "is_shape_wp",
     ],
+    defaults=[False],
```

Comment:
That is not right; you need to put a default for every field. E.g. `[None, [], False]`.

Reply:
Thanks, sure, I changed it to [`None,[], False]`, I thought we only needed the default value for the `is_shape_wp`.